[2023-04-30 12:05:06,558] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.fact_stage manual__2023-04-30T12:03:10.806501+00:00 [queued]>
[2023-04-30 12:05:06,622] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.fact_stage manual__2023-04-30T12:03:10.806501+00:00 [queued]>
[2023-04-30 12:05:06,623] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 12:05:06,624] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-04-30 12:05:06,625] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-30 12:05:06,720] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): fact_stage> on 2023-04-30 12:03:10.806501+00:00
[2023-04-30 12:05:06,732] {standard_task_runner.py:52} INFO - Started process 1154 to run task
[2023-04-30 12:05:06,741] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'fact_stage', 'manual__2023-04-30T12:03:10.806501+00:00', '--job-id', '295', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmp246urst1', '--error-file', '/tmp/tmpxnc7k0s2']
[2023-04-30 12:05:06,744] {standard_task_runner.py:80} INFO - Job 295: Subtask fact_stage
[2023-04-30 12:05:07,333] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.fact_stage manual__2023-04-30T12:03:10.806501+00:00 [running]> on host 295d97741f96
[2023-04-30 12:05:08,295] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=fact_stage
AIRFLOW_CTX_EXECUTION_DATE=2023-04-30T12:03:10.806501+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-30T12:03:10.806501+00:00
[2023-04-30 12:05:08,416] {base.py:68} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-30 12:05:09,302] {base.py:68} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-04-30 12:05:10,490] {python.py:173} INFO - Done. Returned value was: None
[2023-04-30 12:05:10,644] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=fact_stage, execution_date=20230430T120310, start_date=20230430T120506, end_date=20230430T120510
[2023-04-30 12:05:10,733] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-30 12:05:11,084] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
